edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -2470,509 +2472,432 @@ QueryBlockScaleGraphFragmentController = Class
-
-        var editorElement = new Element("div", {
-          className : "editCommentEditor"
+        
+        var newCommentAnchor = new Element("a", {
+          id : "comment." + jsonRequest.commentId
         });
-        var textEditorElement = new Element("textarea", {
-          name : "commentEditor." + commentId
-        }).update(commentText);
-        var saveButtonElement = new Element("input", {
-          type : "button",
-          className : "formButton",
-          value : getLocale().getText("query.comment.saveCommentButton")
+        
+        var newCommentHideShowButton = new Element("div", {
+          className : "queryCommentShowHideButton hideIcon"
         });
+        
+        newCommentHeader.appendChild(newCommentDate);
+        _this._createCommentLinks(newCommentMeta);
+        newComment.appendChild(newCommentAnchor);
+        newComment.appendChild(newCommentHideShowButton);
+        newComment.appendChild(newCommentHeader);
+        newComment.appendChild(newCommentWrapper);
+
+        newCommentWrapper.appendChild(new Element("input", {
+          type : "hidden",
+          name : "commentId",
+          value : jsonRequest.commentId
+        }));
+        
+        newCommentWrapper.appendChild(new Element("input", {
+          type : "hidden",
+          name : "queryPageId",
+          value : jsonRequest.queryPageId
+        }));
+
+        newCommentWrapper.appendChild(newCommentText);
+        newCommentWrapper.appendChild(newCommentMeta);
+
+        newCommentWrapper.appendChild(childrenContainer);
+
+        var childrenParent = $('queryCommentChildren.' + parentCommentId);
+        if (childrenParent) {
+          childrenParent.appendChild(newComment);
+        }
+            
+        Event.observe(newCommentHideShowButton, "click", _this._toggleCommentShowHideButtonClickListener);
+        Event.stopObserving(newCommentElement.down("input"), 'click', _this._saveCommentReplyClickListener);
+        newCommentElement.remove();
+      }
+    });
+  },
+  
+  _createCommentLink : function(parent, containerClass, linkClass, linkText, listener) {
+    var container = new Element("div", {
+      className : containerClass
+    });
+    
+    var link = new Element("a", {
+      href : "#",
+      className : linkClass
+    });
+    
+    container.appendChild(link.update(linkText));
 
-        commentTextElement.hide();
-        editorElement.appendChild(textEditorElement);
-        editorElement.appendChild(saveButtonElement);
-        commentTextElement.insert({
-          before : editorElement
-        });
+    parent.appendChild(container);
+    Event.observe(link, 'click', listener);
+
+    return container;
+  },
+  
+  _createCommentLinks : function(container) {
+    var g = this._createCommentLink(container, "queryCommentNewComment", "queryCommentNewCommentLink", getLocale().getText("query.comment.commentAnswerLink"), this._commentReplyClickListener);
+    console.log(g);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2228)